### PR TITLE
Hello Pull Request

### PR DESCRIPTION
**Pull request by Luchy-B**
Add HTML, CSS, gitignore file. Also added Linters

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (Nyame-Wolf) [inline .gitignore]: Kindly add the  `test.md` file to the .gitignore. Its part of the project requirements
Screenshot 2022-10-26 at 11 37 32

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @Luchy-B  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great setup of linters. 
- Linters are passing

Congrats on your work so far! :clap:  

## Required Changes ♻️

- [ ] Kindly add a readme following the microverse [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md). A readme explains what your project is about to users, recruiters as well as reviewers.
Its also part of the project requirements 
Screenshot 2022-10-26 at 11 37 32

- [ ] I suggest you put a more descriptive pull request message. Its important to explain to a reviewer what you did in the pull request. An example here is highlight in bullet points what you did for the milestone. An example would be:
  - added a html file with Hello world
  - added a css file ...
  - added a .gitignore with ....
to edit the pull request description click on the three dots pointed to by the arrow on screenshot below
Screenshot 2022-10-26 at 11 43 55


There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Yazino12** [comment]
- Hi @Luchy-B, Please resolve the merge conflicts, and request another code review.
image
- This area should be green and all linter checks should be passing. if you have any further questions reach out on slack **@Yasin Warsame**

File: README.md [hunk 2]
```diff
@@ -2,14 +2,13 @@
 
 # Project Name
 
-> Description the project.
+Microverse Hello Project
```
Comment (christianbiring1) [inline README.md:6]: [OPTIONAL] 
- [ ] Kindly make sure your project has a detailed description telling what the project is about for it's a core standard for a repo to be documented professionally.

File: styles.css [hunk 3]
```diff
@@ -1,3 +1,3 @@
-h1{
-    color: blue;
-}
\ No newline at end of file
+h1 {
+  color: blue;
```
Comment (christianbiring1) [inline styles.css:2]: [OPTIONAL]
- [ ] Kindly make sure you select the element using a class and not the generic `HTML tag` as specified in the project requirement [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color))

**Review by christianbiring1 — APPROVED** [review]
Hi @Luchy-B , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
Iapproveokay

## Highlights
- [ ] All linter checks are green :heavy_check_mark: 
- [ ] Good commit messages :+1: 
## Optional suggestions
- [ ] `I highly recommend you implement them for they take your project to the next level` :rocket: 


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**